src/pages/questionPage.js

```diff
@@ -21,9 +21,42 @@ export const initQuestionPage = () => {
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
 
+  const correctAnswer = currentQuestion.correct
+
+  let correctAnswerNumber = document.createElement('span')
+  correctAnswerNumber.classList.add('counting')
+  correctAnswerNumber.textContent = 0;
+
+  let theNumber = correctAnswerNumber.textContent
+  theNumber = 0;
+  const count = function() {
+    return theNumber +=1;
+  }
+  
+  const correctAnswerMessage = document.createElement('div')
```

Comment:
All the code from here onwards (except for the line `userInterface.appendChild(correctAnswerMessage)` should be extracted to a function in the views file.